src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs

```diff
@@ -102,10 +113,32 @@ private void ToggleExpandSingleLine()
             [BitMessageBarType.Success] = "Completed"
         };
 
-        protected override Task OnParametersSetAsync()
+        private string GetTextRole()
         {
-            messageBarIcon = MessageBarIconName.HasValue() ? MessageBarIconName : IconMap[MessageBarType];
-            return base.OnParametersSetAsync();
+            switch (messageBarType)
+            {
+                case BitMessageBarType.Blocked:
+                case BitMessageBarType.Error:
+                case BitMessageBarType.SevereWarning:
+                    return "alert";
+            }
+            return "status";
```

Comment:
please move this line into the default case of the above switch statement.

Reply:
Done